platform/linux-generic/pktio/subsystem.c

```diff
@@ -15,9 +15,55 @@
 ODP_SUBSYSTEM_DEFINE(pktio_ops, "packet IO operations", SUBSYSTEM_VERSION);
 
 /* Instantiate init and term functions */
-ODP_SUBSYSTEM_FOREACH_TEMPLATE(pktio_ops, init_global, ODP_ERR)
+SUBSYSTEM_FOREACH_TEMPLATE(pktio_ops, init_global, ODP_ERR)
+SUBSYSTEM_FOREACH_TEMPLATE(pktio_ops, term_global, ODP_ERR)
 ODP_SUBSYSTEM_FOREACH_TEMPLATE(pktio_ops, init_local, ODP_ERR)
-ODP_SUBSYSTEM_FOREACH_TEMPLATE(pktio_ops, term_global, ODP_ABORT)
+
+int odp_pktio_ops_init_global(bool continue_on_errors)
```

Comment:
This functionality can be done in odp_pktio_global_init function. This will avoid the changes to modular framework as well.

When we have done additional enhancements to shared memory, this code will be deleted. So, can be part of odp_pktio_global_init without affecting the modular framework.